### DISCUSSION TITLE
How to restore settings that have not been modified？

### DISCUSSION THREAD

**Discussion by vvidlee** [valinet/ExplorerPatcher#1871] [General]
I don't want to change explorer, but it's changed somewhere. Weird like this:
![搜狗截图20230820081020]
![搜狗截图20230820080931](https://github.com/valinet/ExplorerPatcher/assets/12469585/b8f3bd7a-8b41-43c1-afd3-aa79945b56bb)
(https://github.com/valinet/ExplorerPatcher/assets/12469585/ec1e61ca-73c9-4de0-b514-ce078e9643cc)

By the way, is there no refresh button on the end of explorer address bar in the win11 23h2?
Many thanks.

**Comment by pyrates999** [comment]
You're running an insider build and those are not usually tested by the developers.

What's the reason why you're running an insider build?

**Reply by vvidlee** [reply]
want to know how to restore the state without any modification. The explorer remains the same after uninstalling ep, and I don't want to reinstall the system.

**Reply by pyrates999** [reply]
With EP not installed, file explorer should have all changes undone to it.

**Reply by vvidlee** [reply]
> With EP not installed, file explorer should have all changes undone to it.

They make changes in file explorer in win11 23H2. I guess i'll wait for the update of ep until the official stable 23h2 version coming.
